Felienne/hedy
# [FEATURE] Enable filtering on submitted programs

**Description**
In this PR we add a filtering option to the programs page to enable users (and teachers!) to filter on submitted programs. We already supported filtering on level and adventure, but not yet on wether or not a program is submitted.

**Fixes**
This PR fixes #3196.

**How to test**
Make sure you have several programs, both submitted and not. Verify that the filtering option works as expected.
